sample-apps/discount-functions-sample-app/extensions/fixed-product-discount/src/main.rs

```diff
@@ -0,0 +1,229 @@
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+
+mod api;
+use api::*;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Configuration {
+    pub value: f64
+}
+
+impl Configuration {
+    const DEFAULT_VALUE: f64 = 50.0;
+
+    fn from_str(str: &str) -> Self {
+        serde_json::from_str(str).unwrap_or_default()
```

Comment:
I think we've been doing `.expect(<Some message explaining config couldn't be deserialized>)` instead of `.unwrap_or_default()`